### PR TITLE
Update contributing

### PR DESCRIPTION
**Pull request by magdalenafuentes**
- [x] Update contributing with updated checklist and contribution policy
- [x] Add table with currently supported datasets and refactor minor things in `README.md`
- [x] Fix TypeERROR message `jams_utils.py`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,26 @@ This library provides tools for working with common MIR datasets, including tool
...
 
+#### Currently supported datasets
+
+| Dataset       | 
+| ------------- |
+| [Beatles](http://isophonics.net/content/reference-annotations-beatles)  | 
+| [DALI](https://github.com/gabolsgabs/DALI)    | 
+| [GuitarSet](http://github.com/marl/guitarset/) |
+| [GTZAN genre](http://marsyas.info/downloads/datasets.html) |
+| [Ikala](gtzan_genre) |
```
Comment (rabitt) [inline README.md:26]: flag

File: README.md [hunk 2]
```diff
@@ -15,6 +15,26 @@ This library provides tools for working with common MIR datasets, including tool
...
+| [MedleyDB Melody](https://medleydb.weebly.com/) |
+| [MedleyDB Pitch](https://medleydb.weebly.com/) |
+| [MedleyDB Solos](https://mirdata.readthedocs.io/en/latest/source/mirdata.html#module-mirdata.medley_solos_db) |
```
Comment (lostanlen) [inline README.md:29]: Medley-solos-DB is the official name

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -16,12 +16,20 @@ Finally, run tox with `tox`.  All tests should pass!
 
 To add a new dataset loader:
 
-1. Create a script in `scripts/`, e.g. `make_my_dataset_index.py`, which generates an index file. (See below for what an index file is!)
-2. Run the script on the canonical version of the dataset and save the index in `mirdata/indexes/` e.g. `my_dataset_index.json`. (Also see below for what we mean by "canonical"!)
+1. Create a script in `scripts/`, e.g. `make_my_dataset_index.py`, which generates an index file. (See below for what an index file is)
+2. Run the script on the canonical version of the dataset and save the index in `mirdata/indexes/` e.g. `my_dataset_index.json`. (Also see below for what we mean by "canonical")
```
Comment (lostanlen) [inline CONTRIBUTING.md:20]: thank you for having removed the exclamation marks

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -16,12 +16,20 @@ Finally, run tox with `tox`.  All tests should pass!
...
-1. Create a script in `scripts/`, e.g. `make_my_dataset_index.py`, which generates an index file. (See below for what an index file is!)
-2. Run the script on the canonical version of the dataset and save the index in `mirdata/indexes/` e.g. `my_dataset_index.json`. (Also see below for what we mean by "canonical"!)
+1. Create a script in `scripts/`, e.g. `make_my_dataset_index.py`, which generates an index file. (See below for what an index file is)
+2. Run the script on the canonical version of the dataset and save the index in `mirdata/indexes/` e.g. `my_dataset_index.json`. (Also see below for what we mean by "canonical") 
 3. Create a module in mirdata, e.g. `mirdata/my_dataset.py`
 4. Create tests for your loader in `tests/`, e.g. `test_my_dataset.py`
 5. Add your module to `docs/source/mirdata.rst`
 6. Add the module to `mirdata/__init__.py`
+7. Add the module to the table in the `README.md` file, section `Currently supported datasets`
+8. Make sure the dataset appears in `https://github.com/ismir/mir-datasets`
```
Comment (lostanlen) [inline CONTRIBUTING.md:26]: unclear what "make sure" means. What do i do if the dataset doesn't appear there?

File: README.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 # mirdata
-common loaders for mir datasets. Find the API documentation [here](https://mirdata.readthedocs.io/en/latest/).
+common loaders for Music Information Retrieval (MIR) datasets. Find the API documentation [here](https://mirdata.readthedocs.io/en/latest/).
```
Comment (lostanlen) [inline README.md:2]: finally, a definition of what MIR means!! <3

File: mirdata/jams_utils.py [hunk 6]
```diff
@@ -70,7 +70,7 @@ def jams_converter(
             if type(beats) != tuple:
                 raise TypeError(
                     'beat_data should be a list of tuples, '
-                    + 'but is a list of {}'.format(type(beats))
+                    + 'but contains a {} element'.format(type(beats))
```
Comment (lostanlen) [inline mirdata/jams_utils.py:73]: thank you for having changed this

File: CONTRIBUTING.md (hunk 4 above)
Comment (magdalenafuentes) [reply CONTRIBUTING.md]: True, alse decided with @rabitt to remove this step for now in the template, so removing it from here as well

**Pull request merged**